extra/proxy.sh

```diff
@@ -14,18 +14,18 @@ test_cmd() {
 
 _TMP='/tmp'
 # proxy version
-_VER="${1}"
+_VER=$(echo ${1} | grep -Po '\d+.\d+.\d+')
```

Comment:
no

Reply:
Well I can change this to `_VER=${1/v/Lapce }` and the other grep is not needed too.

Reply:
there is no reason to change that line at all

Reply:
Then I think it would be better for lapce-proxy to output a version number with the format expected in this bash script. I will close this one.

Reply:
This is not a bash script... it says right so in the first few lines

Reply:
Also version can be `nightly`